style_guides/scss_style_guide.md

```diff
@@ -15,6 +15,44 @@ Our style guide is an extension of [Sass Guidelines by Hugo Giraudel](https://sa
 The Sass Guidelines site recommends using RBG and HSL values to format colors, but we're using
 hex values.
 
+### Bemify for namespacing
+
+We use the [bemify](https://github.com/franzheidl/bemify) for namespacing the sass into a BEM format. We use `component`, `child`, `modifier` and `state` as our mixin names. We've adjusted the plugin's state mixin so that you need to write out the full selector (`@include state('is-happening')`).
+
+#### Example
+
+```
+// Generates .kuiBtn
+@include component('kuiBtn') {
+  color: white;
+  background: gray;
+  padding: 20px;
+
+  // Generates .kuiBtn__icon
+  @include child('icon') {
+    display: inline-block;
+    margin-right: 4px;
+    color: white;
+
+    // Generates .kuiBtn__icon--danger
+    @include modifier('danger') {
+      color: red;
+    }
+  }
+
+  // Generates .kuiBtn--primary
+  @include modifier('primary') {
+    background: blue;
+  }
+
+  // Generates .kuiBtn.is-loading
+  @include state('is-loading') {
```

Comment:
We're currently formatting our state classes as `kuiButton-isLoading` -- can we use that here too?